projects/client/RabbitMQ.Client/src/client/impl/AsyncConsumerWorkService.cs

```diff
@@ -45,7 +45,7 @@ class WorkPool
             readonly ModelBase _model;
             CancellationTokenRegistration _tokenRegistration;
             volatile TaskCompletionSource<bool> _syncSource = new TaskCompletionSource<bool>(TaskCreationOptions.RunContinuationsAsynchronously);
-            private Task _task;
+            private Task _worker;
```

Comment:
@danielmarbach `_worker` is not really used anywhere, though. What does keeping it in an instance variable accomplish?

Reply:
Nothing until we reimplement the graceful shutdown as discussed in your PR